src/generateSummary.ts

```diff
@@ -0,0 +1,53 @@
+import * as fs from 'fs';
```

Comment:
This file doesn't have to be a part of the source code and published to npm because it's only used to generate a summary file on GitHub

Reply:
I agree that it doesn't need to be published to npm but it should be part of the source code so others can generate the summary table if they add new chains.

Reply:
What @mmv08 means is that it should not be in the `src` folder.

Reply:
Done: https://github.com/safe-global/safe-deployments/pull/228/commits/65be0d482d35a984599c4998492df2e5a1ce4204